### PR TITLE
update languages

### PR DESCRIPTION
**Pull request by amazingandyyy**

### CODE REVIEW

File: src/scenes/App/components/Loader/index.js [hunk 1]
```diff
@@ -49,6 +49,7 @@ export default class Loader extends Component {
             <Typography variant="display1" className={classes.loaderPercent}>{syncPercent || 0}</Typography>%
             <p>
               <FormattedMessage id="str.blockSync" defaultMessage="Blockchain syncing." />
+              &nbsp;
```
Comment (amazingandyyy) [inline src/scenes/App/components/Loader/index.js]: this is a `space`

File: src/languageProvider/locales/en_US.json [hunk 2]
```diff
@@ -1,7 +1,8 @@
 {
+  "activitiesTab.Finalize": "Finalize",
   "activitiesTab.History": "Activities History",
   "activitiesTab.Set": "Result Setting",
-  "balance.info": "If you would like to encrypt, backup, or restore your wallet, please do this from the Qtum Wallet. In the application menu, there is a \"Launch Qtum Wallet\" option.",
+  "balance.info": "If you would like to encrypt, backup, or restore your wallet, download Qtum Wallet to do so(click the option of \"Launch Qtum Wallet\" in the application menu).",
   "bottomBar.blockNum": "Current Block Number",
```
Comment (henryzbw) [inline src/languageProvider/locales/en_US.json]: it's not download, just directly open wallet.

File: src/languageProvider/locales/zh-Hans.json [hunk 3]
```diff
@@ -2,7 +2,7 @@
   "activitiesTab.Finalize": "结束",
   "activitiesTab.History": "历史记录",
   "activitiesTab.Set": "设置结果",
-  "balance.info": "如果您想加密，备份或恢复您的钱包，请在Qtum钱包中进行。在程序菜单栏，有一个启动Qtum钱包选项。",
+  "balance.info": "如果您想加密、备份或恢复您的钱包，请使用Qtum钱包中进行完成（在程序菜单栏，有一个「启动Qtum钱包选项」）",
```
Comment (henryzbw) [inline src/languageProvider/locales/zh-Hans.json]: 病句

File: src/languageProvider/locales/zh-Hans.json [hunk 4]
```diff
@@ -42,8 +42,8 @@
   "create.datePast": "时间不能为过去",
   "create.dateTime": "选择日期和时间",
   "create.dialogTitle": "创建事件",
-  "create.duplicateOutcome": "重复结果不被允许。",
-  "create.escrowNoteDesc": "您需要存入{amount}BOT押金以用于创建事件，您可以在提现环节取出。",
+  "create.duplicateOutcome": "不允许重复结果。",
+  "create.escrowNoteDesc": "您需要存入{amount}BOT押金以用于创建事件，您可以在提现模块取出。",
```
Comment (henryzbw) [inline src/languageProvider/locales/zh-Hans.json]: it's not 模块， should be `阶段` or something translated from `phase`

File: src/languageProvider/locales/zh-Hans.json [hunk 5]
```diff
@@ -95,20 +95,20 @@
   "navbar.botCourt": "BOT法庭",
   "navbar.qtumPrediction": "QTUM预测",
   "oracle.betStartTimeDisabledText": "下注开始时间还没到",
-  "oracle.cOracleDisabledText": "您不是此项目的结构设定者, 您必须等到他设置结果, 或者直到公开设置结果开始.",
-  "oracle.consensusThreshold": "共识阈值",
-  "oracle.enterAmountDisabledText": "请输入一个有效金额.",
+  "oracle.cOracleDisabledText": "您不是此项目的结果设定者, 您必须等到他设置结果, 或公开设置结果开始。",
+  "oracle.consensusThreshold": "共识门槛",
```
Comment (henryzbw) [inline src/languageProvider/locales/zh-Hans.json]: keep it as 共识阈值

File: src/languageProvider/locales/zh-Hans.json [hunk 6]
```diff
@@ -95,20 +95,20 @@
...
+  "oracle.enterAmountDisabledText": "请输入一个有效金额",
   "oracle.eventUnconfirmed": "这个已创建的事件还未确认，您在它确认之前不能进行操作。",
   "oracle.finalizingExplanation": "任何人都可以终结这个结果. 这将会结束仲裁阶段并且设置结果为上一轮的结果，赢家可以提现他们的奖励一旦事件最终结束。",
-  "oracle.maxVoteText": "您只能为任意结果最多投票至共识阈值，当前最大投票额度是{amount}BOT.",
+  "oracle.maxVoteText": "您只能为任意结果最多投票至共识门槛，当前最大投票额度是{amount}BOT。",
```
Comment (henryzbw) [inline src/languageProvider/locales/zh-Hans.json]: keep it as 共识阈值

File: src/languageProvider/locales/zh-Hans.json [hunk 7]
```diff
@@ -95,20 +95,20 @@
...
   "oracle.optionIsPrevResult": "该选项在之前已被选为结果",
-  "oracle.selectResultDisabledText": "请点击并选择其中一个结果选项.",
-  "oracle.setResultExplanation": "设置结果要求投入共识阈值数量。",
+  "oracle.selectResultDisabledText": "请选择其中一个结果选项。",
+  "oracle.setResultExplanation": "设置结果要求投入共识门槛数量。",
```
Comment (henryzbw) [inline src/languageProvider/locales/zh-Hans.json]: keep it as 共识阈值

File: src/languageProvider/locales/zh-Hans.json [hunk 8]
```diff
@@ -160,17 +160,17 @@
   "str.m": "分",
   "str.minutes": "分钟",
   "str.name": "名称",
-  "str.newBlock": "新区块到达，您可以刷新以获得更多事件。",
+  "str.newBlock": "新区块到达，您可以刷新以获得最新事件。",
```
Comment (henryzbw) [inline src/languageProvider/locales/zh-Hans.json]: remove this string from all language file. it's useless.

**Pull request merged**